src/query/dblink_scan.c

```diff
@@ -542,9 +542,40 @@ dblink_bind_param (int stmt_handle, VAL_DESCR * vd, DBLINK_HOST_VARS * host_vars
   return S_SUCCESS;
 }
 
+int
+dblink_end_tran (int conn_handle, bool is_abort)
+{
+  int rc;
+  T_CCI_ERROR err_buf;
+
+  if (is_abort)
+    {
+      rc = cci_end_tran (conn_handle, CCI_TRAN_ROLLBACK, &err_buf);
+    }
+  else
+    {
+      rc = cci_end_tran (conn_handle, CCI_TRAN_COMMIT, &err_buf);
+    }
```

Comment:
```suggestion
      rc = cci_end_tran (conn_handle, is_abort ? CCI_TRAN_ROLLBACK : CCI_TRAN_COMMIT, &err_buf);
```
다음과 같이 간단하게 작성할 수도 있을 거 같습니다.

Reply:
수정했습니다. 커멘트 감사합니다.